Framework/Core/CMakeLists.txt

```diff
@@ -127,6 +128,7 @@ o2_add_library(Framework
                                      ${BOOST_FILESYSTEM}
                                      arrow::gandiva_shared
                                      LibUV::LibUV
+                                     PkgConfig::ZeroMQ
```

Comment:
Why is this needed at all?

Reply:
DataInspector files use the ZeroMQ library.